ansible/Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM fedora:28
 
 LABEL name="contrainfra/ansible-executor" \
       maintainer="rnester@redhat.com, cmays@redhat.com, ari@redhat.com" \
-      version="0.0.5" \
+      version="2.7.1" \
```

Comment:
Shouldn't this be the version of our container or should it align with the version of ansible?
@joejstuart thoughts

Reply:
100% agree the FROM defines the fedora container and dnf will install whatever version is on that version at the time the container is prepared.  The version should correspond to our version of this ansible-executor container that was what I thought the version was used for.

Reply:
ack, me and Joe spoke about it, but I forgot to check back abt naming. We can just put like 2 (just to  kinda denote we are at ansible 2.x for now and maybe later switch to 3 for 3.x, just a thought..)

Reply:
will need to change here too likewise btw - https://github.com/CentOS-PaaS-SIG/contra-env-infra/pull/14/files, we ll set it to whichever is decided or just abandon this :)

Reply:
The latest right now is at v1.1.2, so just needs to be something greater than that.